packages/api/src/ontology/ObjectOrInterface.ts

```diff
@@ -16,11 +16,28 @@
 
 import type { InterfaceDefinition } from "./InterfaceDefinition.js";
 import type { ObjectTypeDefinition } from "./ObjectTypeDefinition.js";
+import type { SimplePropertyDef } from "./SimplePropertyDef.js";
 
 export type ObjectOrInterfaceDefinition =
   | ObjectTypeDefinition
   | InterfaceDefinition;
 
+export namespace ObjectOrInterfaceDefinition {
+  export type WithRdp<
```

Comment:
Given how long `ObjectOrInterfaceDefinition.WithRdp` is we should probably just shorten this special one to `WithRdp`